src/endorse/Bukov2/optimize_configs.py

```diff
@@ -268,7 +268,7 @@ def export_vtk_bh_chamber_set(cfg_file, bh_pk_ids, fname, plot=False):
     all_packer_coords, bounds = bh_set.points(cfg)
 
     def t_for_idx(bi, idx):
-        return (idx - bh_set.boreholes[bi].start[0]) / bh_set.boreholes[bi].unit_direction[0]
```

Comment:
Use current config to get correct sign: sim_sfg.active_zk.invert_xy

Reply:
review the other changes as well.

Reply:
Done.